src/js/components/TaskDetailComponent.jsx

```diff
@@ -38,6 +38,29 @@ var TaskDetailComponent = React.createClass({
     );
   },
 
+  getTaskEndpoints: function () {
+    var props = this.props;
+    var task = props.task;
+    if (!task.ports) {
+      return "None";
+    }
+
+    let endpoints = task.ports.map((port) => {
+      let endpoint = `${task.host}:${port}`;
+      return (
+        <li key={endpoint} className="overflow-ellipsis">
+          <a href={endpoint} target="_blank">{endpoint}</a>
```

Comment:
Linking on this is a little bit weird because we don't know the protocol.
But the link needs to have a protocol. 
So if we link, let's add 'http://', otherwise it isn't working. FF doesn't understand the protocol..